admin/tests/unit/services/access-control-test.js

```diff
@@ -44,4 +44,50 @@ module('Unit | Service | access-control', function (hooks) {
       });
     });
   });
+
+  module('#hasAccessToCertificationActionsScope', function () {
+    test('should be true if user is Super admin', function (assert) {
```

Comment:
suggestion `should be true if admin member is Super admin` pour enlever l'ambiguité entre un admin member et un autre type d'user